keps/sig-scheduling/3243-respect-pod-topology-spread-after-rolling-upgrades/README.md

```diff
@@ -659,13 +659,18 @@ feature flags will be enabled on some API servers and not others during the
 rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
+It won't impact already running workloads and it's an opt-in feature. 
+
```

Comment:
at line 643, do we still not have those tests?

Reply:
Updated.
We already have the unit and integration tests.